### PR TITLE
Broadcaster refactoring

### PR DESCRIPTION
**Pull request by fraborg**
# Description
This PR contains a refactoring in the broadcaster module.

- [x] Remove no more needed network parameters
The network parameters are now independent from Advertisement, so the broadcaster doesn't need them anymore. This information was received from the peering-request operator, therefore that has been updated too.
- [x] Thread-safe Advertisement creation
The Advertisement can be created by 2 threads, the one of the broadcaster or the one of the ClusterConfig watcher. Now a mutex blocks these two threads to create Advertisement concurrently.

## Other changes
- The `liqo-configmap` is not used anymore, thereofre it has been deleted from helm chart
- Minor bugfix in `tunnelEnpdpointCreator` manifest

### CODE REVIEW

File: deployments/liqo_chart/templates/configmap.yaml [hunk 1]
```diff
@@ -8,5 +8,4 @@ data:
   clusterID: "placeHolder"
   podCIDR: {{ .Values.podCIDR }}
   serviceCIDR: {{ .Values.serviceCIDR }}
-  gatewayPrivateIP: "192.168.1.1" #placeholder, needed by broadcaster. soon will be removed
   gatewayIP: "placeholder"
```
Comment (fraborg) [inline deployments/liqo_chart/templates/configmap.yaml:11]: is this still needed?

Comment (palexster) [reply deployments/liqo_chart/templates/configmap.yaml:11]: Nope. @alacuku ?

Comment (alacuku) [reply deployments/liqo_chart/templates/configmap.yaml:11]: I think we can remove the whole configmap. The network configuration has been move to the `clusterconfigs.liqo.io` CRD.

**Review by alacuku — APPROVED** [review]

**Review by mLavacca — APPROVED** [review]
/approve

**Review by palexster — APPROVED** [review]

**Pull request merged**